messagecard.go

```diff
@@ -229,7 +229,8 @@ func (mcs *MessageCardSection) AddFactFromKeyValue(key string, values ...string)
 // AddImage adds an image to a MessageCard section. These images are used to
 // provide a photo gallery inside a MessageCard section.
 func (mcs *MessageCardSection) AddImage(sectionImage ...MessageCardSectionImage) error {
-	for _, img := range sectionImage {
+
+	for index, img := range sectionImage {
```

Comment:
Do you like `index` or `idx` better as the variable name? I didn't think about that until looking over the diff just now.

Reply:
I prefer something like this:
```
for i := range list {
    doSomething(list[i])
}
```
In case of `range` I choose only the letter `i` and do not use a variable for the value because it causes more memory operations.

But if you use the same naming for the same kind of construct then it is ok for me. Code style, naming and other alignments can be done later when we figure out that some definitions needed etc.

Reply:
> @dasrick: In case of `range` I choose only the letter `i` for index and do not use a variable for the value because it causes more memory operations.

Good point. I am familiar with that approach, but seem to have fallen out of using it. I've rebased/force-pushed my branch so that the PR reflects that form.

> But if you use the same naming for the same kind of construct then it is ok for me. Code style, naming and other alignments can be done later when we figure out that some definitions needed etc.

Understood. I agree that consistency is good and *try* to follow it.

Thank you for your feedback.